packages/calypso-products/src/sort.js

```diff
@@ -32,23 +33,26 @@ function sortProducts( products ) {
 
 	const includedItems = products.filter( isIncludedWithPlan );
 
-	domainItems = difference( products, includedItems );
+	domainItems = products.filter( ( product ) => ! includedItems.includes( product ) );
 	domainItems = domainItems.filter( isDomainProduct );
 	domainItems = Object.entries( groupBy( domainItems, 'meta' ) );
-	domainItems = sortBy( domainItems, function ( pair ) {
+	domainItems = domainItems.sort( function ( pair ) {
 		if ( pair[ 1 ][ 0 ] && pair[ 1 ][ 0 ].cost === 0 ) {
 			return -1;
 		}
 		return pair[ 0 ];
 	} );
 	domainItems = domainItems.map( function ( pair ) {
-		return sortBy( pair[ 1 ], getDomainProductRanking );
+		return pair[ 1 ].sort( getDomainProductRanking );
 	} );
-	domainItems = flatten( domainItems );
+	domainItems = domainItems.flat();
```

Comment:
These two lines could be merged into one `domainItems.flatMap( ( pair ) => ... )`

Reply:
Good point. I haven't used `flatMap` before so it didn't occur to me.